cmr_images/ReadMe.md

```diff
@@ -0,0 +1,25 @@
+# Cardiac MR Image (Short-Axis) Dataset for Pulmonary Arterial Hypertension Diagnosis
```

Comment:
CMR should not be the top level structure. At least medical_images or else.

Reply:
We should think of a better name for this dataset so others can refer to like those other popular dataset.

Reply:
> CMR should not be the top level structure. At least medical_images or else.

under image

Reply:
> We should think of a better name for this dataset so others can refer to lik

**ShefPAH179**